printer_mods/README.md

```diff
@@ -55,6 +55,7 @@ like so:
 |BlueBear|[Deck Panel Backpack](./BlueBear/Deck_Panel_Backpack)|A mounting alternative if you don't want to drill your deck panel | :heavy_check_mark: | :x: | :x: |:x:|
 ||[Clippable Wago 221 Mounts](./BlueBear/Wago_221_mount)|Mounts for Wago 221 to screw or clip-in on 1515 extrusions | :heavy_check_mark: | :x: | :x: |
 ||[Ziptie Mount](./BlueBear/Ziptie_Mount)|Clippable Ziptie hooks for 1515 extrusions| :heavy_check_mark: | :x: | :x: |
+| bobbleheed | [Magnetic_Panels](./bobbleheed/Magnetic_Panels) | Magnetic corner clips to allow for easy removal and install of panels | :x: | :grey_question: | :heavy_check_mark: | :x: |
```

Comment:
Please remove the underscore from the name in square brackets.  Instead of

    [Magnetic_Panels](./bobbleheed/Magnetic_Panels)

do

    [Magnetic Panels](./bobbleheed/Magnetic_Panels)

Reply:
I've just made the changes.  Although the overhang is just the recommended 0.4mm chamfer on faces touching the build plate.